src/components/screens/management-screen.tsx

```diff
@@ -115,6 +161,23 @@ const ManagementScreen: React.FC = () => {
     setSelectedPersonWithTotalTime(personWithTotalTime);
   };
 
+  /**
+   * Handler for billable hours update dialog
+   */
+  const handleClickOpen = () => {
+    setBillableHoursUpdate(true);
+  };
+
+  /**
+   * Handler for new billable percentage
+   * 
+   */
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = event.target.value;
+    const newValueNumber = Number(newValue);
+    setNewBillablePercentage(newValueNumber);
+  };
```

Comment:
This can be onelined:
```suggestion
  const handleChange = (({ target }): React.ChangeEvent<HTMLInputElement>) => setNewBillablePercentage(Number(target.value));
```

Reply:
Done